### PR TITLE
Removed 'Record Audio' from Image Picker.

### PR DESCRIPTION
**Pull request by jmkulwik**

### CODE REVIEW

**Comment by jmkulwik** [comment]
Notes: I spent some time investigating adding a droplet UI test. I couldn't find an established way to do so. If anyone knows of a way we do this, please let me know. Otherwise, I'll add that to the productivity backlog.

Old:
image

New:
image

File: apps/src/code-studio/components/SoundPicker.jsx [hunk 1]
```diff
@@ -74,23 +74,17 @@ export default class SoundPicker extends React.Component {
     };
 
     let modeSwitch;
-    let title = (
-      <p>
-        {this.props.assetChosen ? "Choose Sounds" : "Manage Sounds"}
-      </p>
-    );
+    let title = (<p>Choose Sounds</p>);
```
Comment (epeach) [inline apps/src/code-studio/components/SoundPicker.jsx]: Nit: Can we i18n 'Choose Sounds' while we're editing?

**Review by epeach — APPROVED** [review]
LGTM! Good catch that we don't need the extra conditionals in SoundPicker.js

One small i18n nit.

File: apps/src/code-studio/components/SoundPicker.jsx (hunk 1 above)
Comment (jmkulwik) [reply apps/src/code-studio/components/SoundPicker.jsx]: Done!

**Pull request merged**